modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/FDSDataRenderers/StatusDataRenderer.tsx

```diff
@@ -6,7 +6,7 @@
 import classNames from 'classnames';
 import React from 'react';
 
-export default function objectDefinitionStatusDataRenderer({
```

Comment:
Why did you rename `objectDefinitionStatusDataRenderer` to `StatusDataRenderer`? Brian might prefer the former.

Reply:
Because the objectDefinitionStatusDataRenderer was only being consumed inside the objectDefinition and the status needs to be consumed in other components outside the objectDefinition, that's why I created the StatusDataRenderer.